### ISSUE TITLE
Recording the current

### ISSUE THREAD

**Issue by Amenty** [NeuralEnsemble/PyNN#565]
From Neo segments it is possible to record certain variables, and gsyn_exc and gsyn_inh among them. Please make the current recordable as well.

**Comment by apdavison** [comment]
Do you mean the synaptic current?

**Comment by Amenty** [comment]
yes, sorry for not mentioning it from the start

**Comment by apdavison** [comment]
This will be a bit tricky to implement for conductance-based synapses - it will probably mean that we have to record `gsyn` and `v` and then calculate the synaptic current from these - but it is a good idea.

**Comment by Amenty** [comment]
Yes, it is exactly how I calculated summed current from my output. This is a good approximation of LFPs.

**Comment by Amenty** [comment]
Thank you for the reply. That is exactly how I have post-processed it. It was useful as an approximation of LFPs.
